pkg/virt-handler/migration-proxy/migration-proxy.go

```diff
@@ -320,7 +335,7 @@ func (m *migrationProxy) createTcpListener() error {
 	var err error
 	if m.serverTLSConfig != nil {
 		listener, err = tls.Listen("tcp", fmt.Sprintf("%s:%d", m.tcpBindAddress, m.tcpBindPort), m.serverTLSConfig)
-	} else if strings.Contains(m.tcpBindAddress, "127.0.0.1") {
+	} else if strings.Contains(m.tcpBindAddress, "127.0.0.1") || strings.Contains(m.tcpBindAddress, "[::1]") {
```

Comment:
I'd encapsulate "if strings.Contains(m.tcpBindAddress, "127.0.0.1") || strings.Contains(m.tcpBindAddress, "[::1]")" in a method named `isLocallyBound` or `isBoundToLocalhost` or something like that.